<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWaveData</class>
 <widget class="QWidget" name="frmWaveData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="WaveData" name="waveData1" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>482</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="WaveData" name="waveData2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>482</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="WaveData" name="waveData3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>482</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WaveData</class>
   <extends>QWidget</extends>
   <header>wavedata.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
